src/main/webapp/queue/ta.html

```diff
@@ -66,13 +66,25 @@
       
       fetch(request);
 
+      getWorkspace(params);
+
       getQueue();
     }
 
+    function getWorkspace(params){
```

Comment:
If this function is used on both the TA and student page consider moving it to a js file that can be imported by both pages.

Reply:
They both do slightly different things!